src/main/io/serial.c

```diff
@@ -211,11 +212,13 @@ bool isSerialConfigValid(serialConfig_t *serialConfigToCheck)
                 return false;
             }
 
-            if (!(portConfig->functionMask & FUNCTION_MSP)) {
```

Comment:
I'm not sure I understand what you mean as I only need to check this specific combination in case of bitCount > 1.
See further, the port sharing is also filtered in blackbox_io.c.